tests/UI/pages/BO/customerService/merchandiseReturns/index.js

```diff
@@ -0,0 +1,35 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+class MerchandiseReturns extends BOBasePage {
+  constructor() {
+    super();
+
+    this.pageTitle = 'Merchandise Returns •';
+    this.successfulUpdateMessage = 'The settings have been successfully updated.';
+
+    // Selectors
+    // Options
+    this.generalForm = '#order_return_fieldset_general';
+    this.enableOrderReturnLabel = toggle => `${this.generalForm} label[for='PS_ORDER_RETURN_${toggle}']`;
```

Comment:
This is an ugly selector.

Reply:
Why ? all our switch buttons selectors are like this one

Reply:
That's not really a problem and I guess Boubker will refactor it at some point in the future. Don't worry.